app/scripts/directives/questionToolbar.js

```diff
@@ -170,16 +170,18 @@ angular
           },
           {
             id: 'campusQuestion',
-            defaultTitle: 'Campus',
+            defaultTitle:
+              'Campus- Type the full name of your campus and choose from the list that appears',
             defaultProfile: 'CAMPUS',
-            iconClass: 'fa-graduation-cap',
+            iconClass: 'fa-university',
             name: 'Campus',
+            defaultExportFieldTitle: 'Campus',
```

Comment:
This is more of a question. What is the need of `defaultExportFieldTitle`, it doesn't seem to do anything.

Reply:
Since the default title for the question is so long, the product owner requested to automatically change the title to something shorter when it is exported.